brainy-ideas/readAloud-valence-jess/readme.md

```diff
@@ -0,0 +1,145 @@
+
+**Design:**
+
+Prior to the experimental task, participants complete several questionnaires:
+- Demographic questionnaire
+- Assessment of reading skill (TBD)
+- Brief Mood Introspection Scale (BMIS)
+- Emotion Regulation Questionnaire (ERQ)
+- Interpersonal Emotion Regulation Questionnaire (IERQ)
+- Beck Depression Inventory (BDI-II)
+- State-Trait Anxiety Inventory (STAI)
+- Affective Reactivity Index (ARI)
+- Interpersonal Reactivity Index (IRI)
+
+
+After completion of the questionnaires, participants are prompted to speak briefly (~1 minute) on any topic; this brief narrative is recorded.  Ostensibly, the purpose of this is to test and calibrate the online recording system.  For the initial analyses, this brief narrative does not serve any specific purpose; however, future analyses may be able to correlate qualities of the narrative with the BMIS measurement.
+
+The main study task involves the reading aloud of 21 short texts (~200 words each) that are designed to evoke a certain mood state in the reader (positive/negative) at the start of the passage, but include a single valence "switch point" where the mood of the text suddenly reverses. (Examples: [pos>neg passage](https://github.com/NDCLab/brainBox/blob/rwe-valence-jess/brainy-ideas/rwe-valence-jess/stimuli/pos-neg_dolphins.txt) and [neg>pos passage](https://github.com/NDCLab/brainBox/blob/rwe-valence-jess/brainy-ideas/rwe-valence-jess/stimuli/neg_pos_dolphins.txt).)  Seven blocks of texts, with each block including three texts, are presented back-to-back in such a way that "switch points" are passage-internal (not between passages) and the total amount of "positive" text is equal to the total amount of "negative" text.  Participants are instructed to read the texts aloud at a comfortable speed, articulating clearly and naturally as if they were rehearsing for a documentary voiceover.  Between each block, a "break" is provided during which the participant is prompted to speak on any topic (similar to the initially recorded narrative) and in keeping with the ecological validity of the study: people typically "chat" when they take breaks between regular life tasks.  The primary functions of these breaks are to give the participant a rest period from the tasking activity of reading aloud and to put the participant back in control of their own mood state before the next block.  A secondary function, if future analyses are able to correlate narrative qualities with the initial BMIS measurement, is to offer the possibility of identifying more dynamic changes in mood during the reading aloud task.
```

Comment:
I really think we need to dig into this idea of what to do during the blocks further. I am very hesitant about the "chat" idea, as this will lead to lots of noise in the data. Yes, you want ecological validity, but baby steps... The study design as a whole is already much, much more ecologically valid. Also, it is important to keep in mind that if you require chatting during the break, not everyone is an extrovert and actually likes to do so. I really think we should have something more controlled between the breaks, or even just nothing at all. But, I think we can discuss this more in the project repo, I dont think this needs to be hashed out here in brain box.

Reply:
meta comment: we should provide some language in the brainbox readme to provide guidelines for what state a project has to be in to be "approved". Certainly not all the little experimental details. But there needs to be the basic structure, which directly tests well worded hypotheses, and there needs to be initial work put into operationalizing ivs/dvs. Basically the state this project is now at (which is great) :)

Reply:
Do you have any ideas for what would serve as a good, neutral break?  I've been wracking my brain, but haven't come up with anything that feels appropriate.

RE: meta comment: I agree that we should have some indication of an appropriate state before approval.  In our case, I'd like to really get the main components of the task planned: what lexical decision task we're doing, how we're doing breaks, etc.  Then all the finer details (like specifics of the stimuli, etc.) can be hashed out later.  So, for now, I'm not going to merge.

Also, my own meta comment: Is the plan, when we do merge, that this folder lives permanently in brainBox, we close the brainBo issue, and the conversation picks back up over in the new project repo?

Reply:
I adjusted the break idea to be more neutral.

For our meta-discussion, we still need to determine:

- guidelines for what constitutes an adequate level for approval to "graduate" from brainBox (although this may also be something that you want to maintain at your own discretion, as you might want, say, a strong undergrad to plan further than a fifth year PhD?).
- What remains in brainBox after approval and merge of the idea branch.  I would now suggest that the project lead move all the materials over to the new repo and leave, in the subdirectory on brainBox, a simple readme that indicates the last commit on brainBox and date of transfer to a separate repo.  I think this will keep brainBox clean, with just a single folder (containing only a readme) for all "graduated" ideas, but enable track back through the brainBox history.

Let me know your thoughts on these and I will use them to guide my readAloud-valence merge here, as well as update brainBox guidance itself!

Reply:
Or perhaps we leave the main readme that outlines all the study ideas instead?